lib/health_cards/qbp_client.rb

```diff
@@ -0,0 +1,217 @@
+# frozen_string_literal: true
+
+require 'savon'
+require 'ruby-hl7'
+require 'faraday'
+require_relative 'qpd'
+
+module HealthCards
+  # Send, receive, and translate HL7 V2 messages from the QBP client of the IIS sandbox
+  module QBPClient # rubocop:disable Metrics/ModuleLength # Disabling since module will be ported and refactored
```

Comment:
I am working on moving this to the iis-shc-issuer repository, so that it will be independent from health cards. 

My thought there was to have one `iis-shc-issuer` folder within `lib/`, containing `qbp_client.rb`, `translator.rb`, `validator.rb`, `generate_shc.rb`, etc., pretty much everything involved in the workflow from entering patient information to receiving a health card back from the IIS. I'm open to talking more about the structure of this in a collab session soon.